ipaplatform/redhat/authconfig.py

```diff
@@ -85,10 +87,16 @@ def execute(self, update=True):
             self.add_option("update")
 
         args = self.build_args()
-        ipautil.run(["/usr/sbin/authconfig"] + args)
+        try:
+            ipautil.run([paths.AUTHCONFIG] + args)
+        except ipautil.CalledProcessError:
+            print("Failed to execute authconfig command")
 
     def backup(self, path):
-        ipautil.run(["/usr/sbin/authconfig", "--savebackup", path])
+        try:
+            ipautil.run([paths.AUTHCONFIG, "--savebackup", path])
+        except ipautil.CalledProcessError:
+            print("Failed to execute authconfig command")
```

Comment:
You could try to `raise ScriptError(msg)` and check if it is properly handled and displayed.